### PR TITLE
Adding ChapmanCheng's file

### PR DESCRIPTION
**Pull request by chapman-cc**
add a new file markdown with yaml

### CODE REVIEW

File: _posts/0000-01-02-ChapmanCheng.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (ChapmanCheng) [inline _posts/0000-01-02-ChapmanCheng.md:5]:
```suggestion
Your text sucks
```

File: _posts/0000-01-02-ChapmanCheng.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
```
Comment (chapman-cc) [inline _posts/0000-01-02-ChapmanCheng.md:3]: Title could be longer

File: _posts/0000-01-02-ChapmanCheng.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
```
Comment (chapman-cc) [inline _posts/0000-01-02-ChapmanCheng.md:2]:
```suggestion
layout: slide.js
```

File: _posts/0000-01-02-ChapmanCheng.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide.js
```
Comment (chapman-cc) [inline _posts/0000-01-02-ChapmanCheng.md]:
```suggestion
layout: slide
```

**Review by chapman-cc — COMMENTED** [review]
Great job

**Pull request merged**